src/Sidebar/Sidebar.tsx

```diff
@@ -8,12 +7,43 @@ import {
   SidebarHeaderStyles,
 } from "./sidebar.style";
 
-const renderableChildrenPropType = PropTypes.oneOfType([
-  PropTypes.arrayOf(PropTypes.node),
-  PropTypes.node,
-]);
+type LinkProps = {
+  iconName: string;
+  active?: boolean;
+  href: string;
+  children?: ReactNode;
+};
+
+type HeaderProps = {
+  logoUrl?: string;
+  logoAltText?: string;
+  href?: string;
+  children?: ReactNode;
+};
+
+type SideBarProps = {
```

Comment:
here also, SideBarProps isn't needed in that case

Reply:
I removed all what you said already before even you comment 😄

Reply:
you can take a look while it finish build and test 😉